src/pkg/leaderElection.go

```diff
@@ -43,8 +44,34 @@ func RunLeaderElection(client *clientset.Clientset, lockName, lockIdentity, lock
 				log.Info().Msgf("Perform Migration")
 				for {
 					log.Info().Msgf("leader is %s", lockIdentity)
-					log.Info().Msgf("Getting replica count...")
-					time.Sleep(5 * time.Second)
+					replicaCount, err := getReplicas()
+					if err != nil {
+						log.Info().Msgf("Failed to get replica count: %v", err)
```

Comment:
```suggestion
						log.Error().Err(err).Msg("Failed to get replica count")
```
1.) Use the `Error` level since this is related to an error
2.) Use the `Err` function for attaching the error object